recipes/pyglm/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pyglm" %}
+{% set version = "2.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyGLM-{{ version }}.tar.gz
+  sha256: 11fd8f409fef081cbf91d93307e4931f39be1e85b3ae12def0897c1cbcd16ecc
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+  run:
+    - libgcc-ng  # [linux]
+    - libstdcxx-ng  # [linux]
```

Comment:
These are added automatically as run_exports by the compiler and should not be explicitly listed.

Reply:
I was getting overlinking errors without these listed though?

Reply:
This shouldn't happen. Please try without them again.